src/index.html

```diff
@@ -15,8 +15,36 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/" class="logo__img"><img src="./images/logo.png" alt="Moyo"></a>
```

Comment:
The `alt` attribute for images should describe the image content. The current `alt` text 'Moyo' is not descriptive of the image content. Consider using a more descriptive alternative, such as the company name or logo description, for example, 'Moyo Logo'.